dev-client/src/screens/LocationScreens/components/SoilIdSections.tsx

```diff
@@ -40,17 +50,38 @@ export const SoilIdDescriptionSection = ({siteId}: SoilIdSectionProps) => {
   );
 };
 
-export const SoilIdMatchesSection = ({siteId}: SoilIdSectionProps) => {
+export const SoilIdMatchesSection = ({siteId, coords}: SoilIdSectionProps) => {
   const {t} = useTranslation();
+  const isSite = !!siteId;
+
+  const locationMatch = LOCATION_BASED_SOIL_MATCH;
+  const dataMatch = DATA_BASED_SOIL_MATCH;
 
   return (
     <ScreenContentSection backgroundColor="grey.200">
       <Row alignItems="center">
         <Heading variant="h6">{t('site.soil_id.matches.title')}</Heading>
         <InfoOverlaySheetButton Header={t('site.soil_id.matches.info.title')}>
-          <TopSoilMatchesInfoContent isSite={!!siteId} />
+          <TopSoilMatchesInfoContent isSite={isSite} />
         </InfoOverlaySheetButton>
       </Row>
+      <InfoOverlaySheet
+        Header={DATA_BASED_SOIL_MATCH.soilInfo.soilSeries.name}
+        trigger={onOpen => (
+          <Button backgroundColor={'background.secondary'} onPress={onOpen}>
+            (Match Goes Here)
```

Comment:
This is a temporary button which will be replaced with an appropriate trigger once the match components are integrated.